### DISCUSSION TITLE
月球只需數小時就能從一團碎片中形成🌛

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#805] [General]
**新聞來源:**
https://www.tam.gov.taipei/News_Content.aspx?n=EF86D8AF23B9A85B&sms=F32C4FF0AC5C2801&s=B225C79F086522AF

**內容大綱:**
英國杜倫大學研究人員使用SWIFT開源代碼模擬了數百種不同的碰撞，以比以往更高的分辨率模擬碰撞的後果，並用來解釋滿足當今地月系統的狀態。
模擬結果顯示，月球是由撞擊後的地球拋射物質和忒亞（Theia）的碎片在幾個小時內形成的，提供了單階段形成理論，為月球的可見特性提供了一個簡潔而優雅的答案，比如具有寬闊和傾斜的軌道、部分熔融的內部結構，以及薄薄的外殼。

**心得感想:**
我覺得有關宇宙天文的一切都好奧妙，月球身為地球的衛星，如果能更深入探討形成背景、碰撞過程發生的事件，或許也能更了解其他衛星的誕生，這種研究之後應用在其他星體的形成，就能讓天文迷對宇宙有更多新見識！
(下面這張圖帥爆了!!)
![image](https://user-images.githubusercontent.com/90883526/196635277-eace922f-b02b-4376-8250-29e28649f6fa.png)

**Comment by moonnight3503** [comment]
利用電腦對大數據進行運算可以產生更加精準的數據，也可模擬更多的可能情況，電腦科學對實驗研究也是不可或缺的一部分呢!

**Comment by ghost** [comment]
電腦可以幫助我們推測各種可能，提供給科學家們有調查驗證的方向，期待未來月球任務能帶回足以證明其可能性的證據。

**Comment by Pietra1226** [comment]
電腦科技進步很有助於我們拓展太空的視野，感覺移居火星是早晚的事，希望有生之年可以離開地球一次。

**Reply by bonny123865** [reply]
移居火星嗎!!?這我倒是沒想過!
感覺也有機會在太陽系之外的星系找到更適合居住的地方欸!

**Comment by evanchou1204** [comment]
用電腦好像不管過去還是未來都慢慢可以精準預測了，好神奇。

**Comment by ru0372572** [comment]
以前只知道月球是碰撞形成的，但居然不用花很多時間ㄟ。隨著科技進步，高分辨率的模擬能導出更貼近事實的結果，幫助我們了解各種成因！

**Comment by Chiway1229** [comment]
科技進展對於宇宙觀測有很大的進步，希望有生之年能看到 發現適合人類居住的新星球的一天

**Reply by bonny123865** [reply]
我也希望能有這樣的一天 就像瓦力的電影一樣
但是不希望地球是被我們破壞到無法居住的時候才找替代環境!

**Comment by ruthchen1020** [comment]
可是一個小時的時間怎麼計算...做成一顆星球不會影響時間的計算嗎

**Comment by ghost** [comment]
感嘆現在科技的進步，使我們能通過電腦運算就能驗證假設的可行性，然可行不代表就是事實，還必須要有物證才能夠證實其真實性，期待未來的宇航員能夠從月球取得更多的樣本，以解開這多年問題。

**Comment by StevenWanglolz** [comment]
真的好感嘆現在的科技，離我們那麼遙遠的東西透過一台電腦就可以計算各種可能性!
